File: Data Structures & Algorithms/longest-repeating-substring-with-replacement/submission-1.py
class Solution:
    def characterReplacement(self, s: str, k: int) -> int:
        S, RNG = {}, 0
        L, R = 0, 0
        MAX = 0

        while R < len(s):
            S[s[R]] = 1 + S.get(s[R], 0)
            MAX=max(MAX, S[s[R]])

            while ((R-L+1)-MAX)>k:
                S[s[L]]-=1
                L+=1
            RNG = max(RNG, R-L+1)
            R+=1
        return RNG

File: Data Structures & Algorithms/longest-consecutive-sequence/submission-1.cs
public class Solution
{
    public int LongestConsecutive(int[] nums)
    {
        int longest = 0;
        HashSet<int> nHash = new HashSet<int>(nums);
        foreach (int num in nums)
        {
            int curr = 0;
            if (nHash.Contains(num+1+curr)) { curr++; }
            longest = Math.Max(curr,longest);
            console.WriteLine(longest);
        }
        return longest;
    }
}